convertRaDecToCgemUnits.py

```diff
@@ -32,14 +32,13 @@ def convertSeconds(self, seconds):
             self.strGotoValue += '0'
         return self.strGotoValue
 
-
-    def highMidLow(self, seconds):
```

Comment:
@RobSteele49 as a sanity check, are we ready to remove this method from the interface?